src/Entity/NextBackupWindow.php

```diff
@@ -25,4 +25,17 @@ class NextBackupWindow extends AbstractEntity
      * @var string
      */
     public $end;
+
+    /**
+     * Build a new backup window instance
+     *
+     * @param \stdClass|array|null $parameters
+     */
+    public function build($parameters)
+    {
+        if (is_null($parameters))
+            return;
```

Comment:
I'd have rebased, but the PR is already in. If @toin0u wants me to rebase and re-submit, I'll go and do it